src/core/worker.js

```diff
@@ -517,6 +517,7 @@ class WorkerMessageHandler {
       "GetOperatorList",
       function wphSetupRenderPage(data, sink) {
         var pageIndex = data.pageIndex;
+        const annotationStorage = data.annotationStorage;
```

Comment:
Since the annotation storage is only used below, we don't need a variable here and can just use `annotationStorage: data.annotationStorage,` below.